### PR TITLE
Update i18n resources

### PR DESCRIPTION
**Pull request by simonleung8**
I sorted the language files and Github is able to show a diff

Just for reference, [this](https://github.com/simonleung8/CF-CLI-i18n-manager) is something I threw together quickly to sort the languages.

### CODE REVIEW

File: cf/i18n/resources/de-de.all.json [hunk 1]
```diff
@@ -2329,7 +2329,7 @@
   },
   {
     "id": "HEALTH_CHECK_TYPE",
-    "translation": "TYP-DER-ZUSTANDSPRÜFUNG"
+    "translation": "TYP DER ZUSTANDSPRÜFUNG"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: Can you make that `TYP_DER_ZUSTANDSPRÜFUNG`, i.e. no spaces?

File: cf/i18n/resources/de-de.all.json [hunk 2]
```diff
@@ -2853,19 +2853,19 @@
...
+    "translation": "Loggregator-Endpunkt fehlt in Konfigurationsdatei"
   },
   {
     "id": "Looking up '{{.filePath}}' from repository '{{.repoName}}'",
     "translation": "Im Repository '{{.repoName}}' nach '{{.filePath}}' suchen"
   },
   {
     "id": "MANIFEST_PATH",
-    "translation": "MANIFESTPFAD"
+    "translation": "MANIFEST_PATH (MANIFESTPFAD)"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: Please don't use brackets in USAGE. Isn't the current translation fine?
```
   cf push [-f MANIFESTPFAD]
```

File: cf/i18n/resources/de-de.all.json [hunk 3]
```diff
@@ -2953,11 +2953,11 @@
   },
   {
     "id": "NEW_NAME",
-    "translation": "NEW_NAME (NEUER NAME)"
+    "translation": "NEUER NAME"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: Please use underscore.

File: cf/i18n/resources/de-de.all.json [hunk 4]
```diff
@@ -2953,11 +2953,11 @@
...
   },
   {
     "id": "NUM_INSTANCES",
-    "translation": "ANZAHL-INSTANZEN"
+    "translation": "ANZAHL INSTANZEN"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: Please use underscore.

File: cf/i18n/resources/de-de.all.json [hunk 5]
```diff
@@ -3133,7 +3133,7 @@
   },
   {
     "id": "OPTIONS:",
-    "translation": "OPTIONEN:"
+    "translation": "Optionen:"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: We had this issue last time. It has to be uppercase.

File: cf/i18n/resources/de-de.all.json [hunk 6]
```diff
@@ -1349,11 +1349,11 @@
   },
   {
     "id": "Custom buildpack by name (e.g. my-buildpack) or Git URL (e.g. 'https://github.com/cloudfoundry/java-buildpack.git') or Git URL with a branch or tag (e.g. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' for 'v3.3.0' tag). To use built-in buildpacks only, specify 'default' or 'null'",
-    "translation": "Angepasstes Buildpack nach Name (z. B. my-buildpack) oder Git-URL (z. B. 'https://github.com/cloudfoundry/java-buildpack.git') oder Git-URL mit Zweig oder Tag (z. B. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' für Tag 'v3.3.0'). Geben Sie zur ausschließlichen Verwendung von integrierten Buildpacks 'default' oder 'null' an."
+    "translation": "Angepasstes Buildpack nach Name (z. B. my-buildpack) oder Git-URL (z. B. 'https://github.com/cloudfoundry/java-buildpack.git') oder Git-URL mit Zweig oder Tag (z. B. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' für Tag 'v3.3.0'). Geben Sie zur ausschließlichen Verwendung von integrierten Buildpacks 'default' oder 'null' an"
   },
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: The German person on my team (and some googling I did) seem to indicate it's more common to not use a space in `z.B.`

File: cf/i18n/resources/de-de.all.json [hunk 7]
```diff
@@ -2909,15 +2909,15 @@
...
   {
     "id": "Maximum amount of memory an application instance can have (e.g. 1024M, 1G, 10G)",
-    "translation": "Maximalwert für den möglichen Speicher einer Anwendungsinstanz (z.B. 1024M, 1G, 10G)"
+    "translation": "Maximalwert für den möglichen Speicher einer Anwendungsinstanz (z. B. 1024M, 1G, 10G)"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: The German person on my team (and some googling I did) seem to indicate it's more common to not use a space in `z.B.`. Why this change?

File: cf/i18n/resources/de-de.all.json [hunk 8]
```diff
@@ -3409,7 +3409,7 @@
   },
   {
     "id": "RESERVED_ROUTE_PORTS",
-    "translation": "RESERVIERTE-ROUTENPORTS"
+    "translation": "RESERVIERTE ROUTENPORTS"
```
Comment (dkoper) [inline cf/i18n/resources/de-de.all.json]: Please use underscore.

File: cf/i18n/resources/es-es.all.json [hunk 9]
```diff
@@ -3257,7 +3257,7 @@
   },
   {
     "id": "Perform a simple check to determine whether a route currently exists or not",
-    "translation": "Realice una comprobación simple para determinar si existe o no en este momento una ruta"
+    "translation": "Realice una comprobación simple para determinar si existe o no en este momento una ruta."
```
Comment (dkoper) [inline cf/i18n/resources/es-es.all.json]: This is the description of the `check-route` command. Please don't add a period at the end of single-sentence command descriptions.

File: cf/i18n/resources/fr-fr.all.json [hunk 10]
```diff
@@ -2024,8 +2024,8 @@
     "translation": "Echec de l'extraction des packs de construction.\n{{.Error}}"
   },
   {
-    "id": "Failed fetching domains for organization {{.OrgName}}.\n{{.Err}}",
-    "translation": "Echec de l'extraction des domaines pour l'organisation {{.OrgName}}.\n{{.Err}}"
+    "id": "Failed fetching domains for organization %s.\n{{.Err}}",
```
Comment (dkoper) [inline cf/i18n/resources/fr-fr.all.json]: This looks suspicious - Are you sure you want to change the message id?

File: cf/i18n/resources/ja-jp.all.json [hunk 11]
```diff
@@ -149,7 +149,7 @@
   },
   {
     "id": "**Attention: Plugins are binaries written by potentially untrusted authors. Install and use plugins at your own risk.**\n\nDo you want to install the plugin {{.Plugin}}? (y or n)",
-    "translation": "**注意: プラグインは必ずしも信頼できない作成者によって書かれたバイナリーです。プラグインのインストールと使用は自らの責任で行ってください。**\n\nプラグイン {{.Plugin}} をインストールしますか? (y または n)"
+    "translation": "**注意: プラグインは必ずしも信頼できない作成者によって書かれたバイナリーです。 プラグインのインストールと使用は自らの責任で行ってください。**\n\nプラグイン {{.Plugin}} をインストールしますか? (y または n)"
```
Comment (dkoper) [inline cf/i18n/resources/ja-jp.all.json]: It is not common in Japanese to add a space after a period. Why this change?

File: cf/i18n/resources/ja-jp.all.json [hunk 12]
```diff
@@ -237,11 +237,11 @@
   },
   {
     "id": "Alias `{{.Command}}` in the plugin being installed is a native CF command/alias.  Rename the `{{.Command}}` command in the plugin being installed in order to enable its installation and use.",
-    "translation": "インストールしようとしているプラグイン内の別名 `{{.Command}}` はネイティブ CF コマンド/別名です。インストールしようとしているプラグインのインストールと使用を可能にするためには、そのプラグイン内の `{{.Command}}` コマンドを名前変更してください。"
+    "translation": "インストールしようとしているプラグイン内の別名 `{{.Command}}` はネイティブ CF コマンド/別名です。  インストールしようとしているプラグインのインストールと使用を可能にするためには、そのプラグイン内の `{{.Command}}` コマンドを名前変更してください。"
```
Comment (dkoper) [inline cf/i18n/resources/ja-jp.all.json]: It is not common in Japanese to add two spaces after a period. Why this change?

File: cf/i18n/resources/ja-jp.all.json [hunk 13]
```diff
@@ -1253,7 +1253,7 @@
   },
   {
     "id": "Creating an app manifest from current settings of app ",
-    "translation": "アプリの現在の設定からアプリ・マニフェストを作成しています"
+    "translation": "アプリの現在の設定からアプリ・マニフェストを作成しています "
```
Comment (dkoper) [inline cf/i18n/resources/ja-jp.all.json]: It is not common in Japanese to add a space at the end of a sentence. Why this change?

File: cf/i18n/resources/ko-kr.all.json [hunk 14]
```diff
@@ -4129,7 +4129,7 @@
   },
   {
     "id": "Terminate the running application Instance at the given index and instantiate a new instance of the application with the same index",
-    "translation": "주어진 색인에서 실행 중인 애플리케이션 인스턴스를 종료하고 애플리케이션의 새 인스턴스를 동일한 색인으로 인스턴스화합니다"
+    "translation": "주어진 색인에서 실행 중인 애플리케이션 인스턴스를 종료하고 애플리케이션의 새 인스턴스를 동일한 색인으로 인스턴스화합니다."
```
Comment (dkoper) [inline cf/i18n/resources/ko-kr.all.json]: This is the description of `restart-app-instance`. Please don't add a period at the end of a command description.

File: cf/i18n/resources/pt-br.all.json [hunk 15]
```diff
@@ -4605,7 +4605,7 @@
   },
   {
     "id": "[global options] command [arguments...] [command options]",
-    "translation": "[opções globais] comando [argumentos...] [opções de comando]"
+    "translation": "[global options] command [arguments...] [command options]"
```
Comment (dkoper) [inline cf/i18n/resources/pt-br.all.json]: We had this issue previous time as well. Please don't revert this translation.

File: cf/i18n/resources/zh-hans.all.json [hunk 16]
```diff
@@ -169,7 +169,7 @@
   },
   {
     "id": "ALIAS:",
-    "translation": "别名:"
+    "translation": "别名: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space after help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 17]
```diff
@@ -985,7 +985,7 @@
...
-    "translation": "CF_NAME update-user-provided-service SERVICE_INSTANCE [-p CREDENTIALS] [-l SYSLOG_DRAIN_URL] [-r ROUTE_SERVICE_URL]\n\n   传递逗号分隔的凭证参数名称以启用交互方式:\n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p \"comma, separated, parameter, names\"\n\n   将凭证参数作为 JSON 传递，从而以非交互方式创建服务:\n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p '{\"key1\":\"value1\",\"key2\":\"value2\"}'\n\n   指定包含 JSON 的文件的路径:\n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p PATH_TO_FILE"
+    "translation": "CF_NAME update-user-provided-service SERVICE_INSTANCE [-p CREDENTIALS] [-l SYSLOG_DRAIN_URL] [-r ROUTE_SERVICE_URL]\n\n   传递逗号分隔的凭证参数名称以启用交互方式: \n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p \"comma, separated, parameter, names\"\n\n   将凭证参数作为 JSON 传递，从而以非交互方式创建服务: \n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p '{\"key1\":\"value1\",\"key2\":\"value2\"}'\n\n   指定包含 JSON 的文件的路径: \n   CF_NAME update-user-provided-service SERVICE_INSTANCE -p PATH_TO_FILE"
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Why introduce spaces before newlines?

File: cf/i18n/resources/zh-hans.all.json [hunk 18]
```diff
@@ -1641,7 +1641,7 @@
   },
   {
     "id": "ENVIRONMENT VARIABLES:",
-    "translation": "环境变量:"
+    "translation": "环境变量: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 19]
```diff
@@ -2137,7 +2137,7 @@
   },
   {
     "id": "GLOBAL OPTIONS:",
-    "translation": "全局选项:"
+    "translation": "全局选项: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 20]
```diff
@@ -2949,7 +2949,7 @@
   },
   {
     "id": "NAME:",
-    "translation": "名称:"
+    "translation": "名称: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 21]
```diff
@@ -3405,19 +3405,19 @@
...
   },
   {
     "id": "RESPONSE:",
-    "translation": "响应:"
+    "translation": "响应: "
   },
   {
     "id": "ROLES:\n",
-    "translation": "角色:\n"
+    "translation": "角色: \n"
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers. It's before a newline anyway.

File: cf/i18n/resources/zh-hans.all.json [hunk 22]
```diff
@@ -4061,59 +4061,59 @@
...
+    "translation": "系统提供的项: "
   },
   {
     "id": "TIMEOUT",
     "translation": "TIMEOUT"
   },
   {
     "id": "TIP:\n",
-    "translation": "提示:\n"
+    "translation": "提示: \n"
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 23]
```diff
@@ -4249,7 +4249,7 @@
   },
   {
     "id": "USAGE:",
-    "translation": "用法:"
+    "translation": "用法: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hans.all.json [hunk 24]
```diff
@@ -4517,7 +4517,7 @@
   },
   {
     "id": "VERSION:",
-    "translation": "版本:"
+    "translation": "版本: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hans.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hant.all.json [hunk 25]
```diff
@@ -2137,7 +2137,7 @@
   },
   {
     "id": "GLOBAL OPTIONS:",
-    "translation": "廣域選項:"
+    "translation": "廣域選項: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hant.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hant.all.json [hunk 26]
```diff
@@ -3133,7 +3133,7 @@
   },
   {
     "id": "OPTIONS:",
-    "translation": "選項:"
+    "translation": "選項: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hant.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hant.all.json [hunk 27]
```diff
@@ -2949,7 +2949,7 @@
   },
   {
     "id": "NAME:",
-    "translation": "名稱:"
+    "translation": "名稱: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hant.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hant.all.json [hunk 28]
```diff
@@ -4249,7 +4249,7 @@
   },
   {
     "id": "USAGE:",
-    "translation": "用法:"
+    "translation": "用法: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hant.all.json]: Please don't add a space at the end of help section headers.

File: cf/i18n/resources/zh-hant.all.json [hunk 29]
```diff
@@ -4517,7 +4517,7 @@
   },
   {
     "id": "VERSION:",
-    "translation": "版本:"
+    "translation": "版本: "
```
Comment (dkoper) [inline cf/i18n/resources/zh-hant.all.json]: Please don't add a space at the end of help section headers.

**Comment by dkoper** [comment]
Closing, this PR introduces too many issues.

File: cf/i18n/resources/ja-jp.all.json (hunk 13 above)
Comment (simonleung8) [inline cf/i18n/resources/ja-jp.all.json]: The original language actually have a space at the end of the sentence
